app.go

```diff
@@ -615,17 +601,32 @@ func (app *app) runShell(s string, args []string, prefix string) cleanFunc {
 				log.Printf("running shell: %s", err)
 			}
 		}()
-	case "$|":
-		return func() {
-			if err := cmd.Wait(); err != nil {
-				log.Printf("running shell: %s", err)
-			}
-			app.nav.renew()
-			if err := app.ui.resume(); err != nil {
-				app.quit()
-				os.Exit(3)
-			}
-		}
 	}
-	return noopCleanFunc
+
+}
+
+func (app *app) runPagerOnText(text io.Reader) {
+	app.nav.exportFiles()
+	app.ui.exportSizes()
+	exportOpts()
+
+	s := envPager
+	args := []string(nil)
+	cmd := shellCommand(s, args)
```

Comment:
You can simplify this:

```go
cmd := shellCommand(envPager, nil)
```

Reply:
Sure, if you prefer. I wanted to make it more obvious how to use this with commands other than a pager, but it's probably not too hard to figure out regardless if we ever need it.